docs/docs/.vuepress/config.js

```diff
@@ -196,6 +346,9 @@ module.exports = {
           "/subprocessors/",
         ],
       },
+      ["/support/", "Support and Community"],
```

Comment:
Can you fetch and merge the latest changes on `master` into this branch? This link has since been changed to https://pipedream.com/support

Reply:
done